felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/BehandlingType.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.familie.kontrakter.felles.tilbakekreving
+
+enum class BehandlingType(val visningsnavn: String) {
+    TILBAKEKREVING("Tilbakekreving"),
+    REVURDERING_TILBAKEKREVING("Tilbakekreving revurdering")
```

Comment:
Må kallende fagsystemer vite om de skal opprette en revurdering av tilbakekrevingsbehandlingen? Eller kan vi utlede det i familie-tilbake?

Reply:
Jeg er usikker om det. Jeg tror familie-tilbake-frontend kaller familie-tilbake når revurdering av tilbakekreving opprettes. Den kan bruke samme kontrakt

Reply:
Opprettelse av revurdering tilbakekreving bør berre være i familie-tilbake-sfæren. Fagsystemet bør ha mulighet for å opprette ein tilbakekrevingsbehandling, men å opprette ein revurdering av tilbakekrevingsbehandling bør gjerast i familie-tilbake.

BehandlingsTypen bør i fagsystemet berre brukes til å vise kva type behandling det er i behandlingsoversikten på fagsaken.

Reply:
sett BehandlingType default verdi til TILBAKEKREVING